dsgrid/config/project_config.py

```diff
@@ -636,16 +636,17 @@ def make_dimension_association_table(self, data_source=None):
                 table = table.filter(f"{ds} = '{data_source}'").drop(ds)
             table_columns.update(table.columns)
 
-        exclude = set((DimensionType.TIME, DimensionType.DATA_SOURCE))
+        exclude = {DimensionType.TIME}
+        if data_source is not None:
```

Comment:
I rather not drop data_source at all in either cases. Having data_source in the table helps with sanity check.